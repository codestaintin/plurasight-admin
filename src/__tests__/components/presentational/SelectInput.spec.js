import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { configure, shallow } from 'enzyme';
import SelectInput from '../../../components/Commons/SelectInput';

configure({ adapter: new Adapter() });

describe('<SelectInput>', () => {
  const props = {
    name: '',
    label: '',
    onChange: jest.fn(() => {}),
    defaultOption: '',
    value: '',
    error: '',
    options: []
  };

  it('renders the <SelectInput> component', () => {
    const tree = shallow(<SelectInput {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
