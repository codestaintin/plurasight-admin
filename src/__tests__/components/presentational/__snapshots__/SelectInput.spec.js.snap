// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SelectInput> renders the <SelectInput> component 1`] = `
<div
  className="form-group"
>
  <label
    htmlFor=""
  />
  <div
    className="field"
  >
    <select
      className="form-control"
      name=""
      onChange={[MockFunction]}
      value=""
    >
      <option
        value=""
      />
    </select>
  </div>
</div>
`;
